openpype/plugins/load/delivery.py

```diff
@@ -222,7 +241,9 @@ def deliver(self):
                     args[0] = src_path
                     if frame:
```

Comment:
The `deliver_sequence` is deprecated and should not be used for representations with `files`. I meant to recalculate the frames here and still use `deliver_single_file`.
```python
        if repre.get("files"):
            src_paths = []
            for repre_file in repre["files"]:
                src_path = self.anatomy.fill_root(repre_file["path"])
                src_paths.append(src_path)
            sources_and_frames = collect_frames(src_paths)

            frames = set(sources_and_frames.values())
            frames.discard(None)
            first_frame = None
            if frames:
                first_frame = min(frames)

            for src_path, frame in sources_and_frames.items():
                # Renumber frames
                if has_renumbered_frame and frame is not None:
                    # Calculate offset between first frame and current frame
                    # - '0' for first frame
                    offset = frame - first_frame
                    # Add offset to new frame start
                    dst_frame = renumber_frame + offset
                    if dst_frame < 0:
                        msg = "Renumber frame has a smaller number than original frame"     # noqa
                        report_items[msg].append(src_file_name)
                        self.log.warning("{} <{}>".format(msg, context))
                        continue
                    frame = dst_frame

                if frame is not None:
                    anatomy_data["frame"] = frame
                new_report_items, uploaded = deliver_single_file(*args)
                report_items.update(new_report_items)
                self._update_progress(uploaded)
```